lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -202,6 +203,17 @@ public IJobSubmissionResult SubmitAndGetJobStatus(JobRequest jobRequest)
         }
 
         /// <summary>
+        /// Returns all the application reports running in the cluster
+        /// </summary>
+        /// <returns></returns>
+        [Unstable("0.16", "Working in progress for rest API id returned")]
+        public async Task<List<IApplicationReport>> GetApplicationReports()
+        {
+            await Task.Delay(0);
+            throw new NotImplementedException();
```

Comment:
Yes, as a sub item. We setup a rule not to allow missing implementations or known issues with out a filed JIRA.